File: executors/node/runner.js
const input = process.env.INPUT || '';
const code = process.env.CODE || '';

if (input) {
    process.stdin.push(input);
}

try {
    eval(code);
} catch (e) {
    console.error(`Error: ${e.message}`);
    process.exit(1);
}

File: executors/python/runner.py
import os
import sys
from io import StringIO

def main():
    code = os.environ.get('CODE', '')
    input_data = os.environ.get('INPUT', '')
    
    if input_data:
        sys.stdin = StringIO(input_data)
    
    try:
        exec_globals = {
            '__name__': '__main__',
            '__file__': 'user_code.py',
        }
        exec(code, exec_globals)
    except Exception as e:
        print(f"Error: {str(e)}", file=sys.stderr)
        sys.exit(1)

if __name__ == "__main__":
    main()